tools/pdf2txt.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+import argparse
 import sys
 from pdfminer.pdfdocument import PDFDocument
```

Comment:
Many rows in pdf2txt.py does not satisfy flake8. See github actions for the specific rows